package com.PeanutJava.OOP.Exception;

public class test3 {
    public static void main(String[] args) {
        System.out.println(6);
        throw new RuntimeException("For no reason.");

    }

}
